junifer/preprocess/bold_warper.py

```diff
@@ -35,8 +36,13 @@ class BOLDWarper(BasePreprocessor):
     ] = [
         {
             "name": "fsl",
-            "optional": False,
-            "commands": ["applywarp"],
+            "optional": True,
```

Comment:
How does this work with the dependency check?

Reply:
In the `BOLDWarper`, the check is optional, but when the actual preprocessor class i.e., `ApplyWarper` and `AntsApplyTransformsWarper` are initialised inside it, the checks are mandatory and fails if the tools are not found.

Reply:
Excellent!